larq/models.py

```diff
@@ -415,9 +435,6 @@ def summary(model, print_fn=None, include_macs=True):
             "`input_shape` argument in the first layer(s) for automatic build."
         )
 
-    if print_fn is None:
-        print_fn = print
-
```

Comment:
I think we still need this.

Reply:
I made it the default value of the arg above so I don't think so?

https://github.com/larq/larq/blob/246a64a4d0ce7f3044cafee58dbb04b42b94879b/larq/models.py#L395

Reply:
Ahh sorry, missed that. I think there was a reason, why I didn't do that in the first place. But I can't remember and CI is happy with it :+1: